mappings/net/minecraft/client/gui/screen/MojangLogoLoadingScreenOverlay.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_425 net/minecraft/client/gui/screen/SplashScreen
+CLASS net/minecraft/class_425 net/minecraft/client/gui/screen/MojangLogoLoadingScreenOverlay
```

Comment:
Not a fan of such a verbose name. `LoadingScreenOverlay` is enough

Reply:
@ramidzkh What about the others? Do I need to reopen a PR?

Reply:
You can just go back to your local yarn, rename this class, save, and use git to commit and push to your branch

Reply:
+1 for `LoadingScreenOverlay` (that was called a `Screen` even though it doesn't inherit from it; it literally wasted half an hour of my life trying to find it :sadtater:)

Reply:
Should probably be like `SplashOverlay` then, see https://github.com/FabricMC/yarn/pull/1992#issuecomment-770912109